modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase

```diff
@@ -0,0 +1,79 @@
+@component-name = "portal-user-management-and-portal-configuration"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Account";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			Accounts.tearDownCP();
+		}
+	}
+
+	@description = "This is a use case for LPS-151522. This asserts saving edits in one permissions tab in Account Roles does not clear related permissions from the other tab."
+	@priority = "5"
+	test CanSavePermissionsWithoutDeletingOthers {
+		task ("Given an account role is created") {
+			Role.openRolesAdmin();
+
+			Navigator.gotoNavTab(navTab = "Account Roles");
+
+			Role.add(roleTitle = "New Account Role");
+		}
+
+		task ("When a permission is saved in 'Define Permissions' tab and another related permission is saved in 'Define Group Scope Permissions' tab") {
+			Role.openRolesAdmin();
+
+			Role.definePermissionCP(
```

Comment:
Same as above, if the issue only reproducible by defining the first permission via UI, we can keep the steps like this, otherwise we can adding it via json.